java/squeek/spiceoflife/foodtracker/commands/CommandFoodList.java

```diff
@@ -74,9 +74,9 @@ public void processCommand(ICommandSender commandSender, String[] args) {
                 final int foodsUntilNextMilestone = progressInfo.foodsUntilNextMilestone();
 
                 commandSender.addChatMessage(new ChatComponentText("" + EnumChatFormatting.BOLD + EnumChatFormatting.DARK_AQUA + player.getDisplayName() + "'s" + EnumChatFormatting.RESET + " food stats:"));
-                commandSender.addChatMessage(new ChatComponentText("Food Eaten: " + foodsEaten));
+                commandSender.addChatMessage(new ChatComponentText("Shanks's worth eaten: " + foodsEaten));
```

Comment:
no `'s`

Reply:
I'm not an english major, I wasn't sure if it should be Shanks, Shanks', or Shanks's.